granary/tests/test_reddit.py

```diff
@@ -76,20 +96,59 @@ def __init__(self, fake_redditor):
         'id': 'tag:reddit.com:ezv3f2',
         'objectType': 'note',
         'published': '2020-02-06T16:51:59Z',
+        'tags': [
+          {'displayName': 'https://www.bonkerfield.org/2020/02/combining-gpt-2-and-bert/',
+          'objectType': 'article',
+          'url': 'https://www.bonkerfield.org/2020/02/combining-gpt-2-and-bert/'},
+          {'displayName': 'https://www.bonkerfield.org/2020/02/reddit-bot-gpt2-bert/',
+          'objectType': 'article',
+          'url': 'https://www.bonkerfield.org/2020/02/reddit-bot-gpt2-bert/'}
+        ],
         'to': [{'alias': '@public', 'objectType': 'group'}],
-        'url': 'https://reddit.com/r/MachineLearning/comments/ezv3f2/p_gpt2_bert_reddit_replier_i_built_a_system_that/'
-      },
-      'url': 'https://reddit.com/r/MachineLearning/comments/ezv3f2/p_gpt2_bert_reddit_replier_i_built_a_system_that/',
-      'verb': 'post'
+        'url': 'https://reddit.com/r/MachineLearning/comments/ezv3f2/p_gpt2_bert_reddit_replier_i_built_a_system_that/'},
+        'url': 'https://reddit.com/r/MachineLearning/comments/ezv3f2/p_gpt2_bert_reddit_replier_i_built_a_system_that/',
+        'verb': 'post'
+      }
+    ],
+    'itemsPerPage': 1,
+    'sorted': False,
+    'startIndex': 0,
+    'totalResults': 1,
+    'updatedSince': False
+  }
+
+COMMENT_OBJECT = {
+  'author': {
+    'description': 'https://bonkerfield.org https://viewfoil.bonkerfield.org',
+    'displayName': 'bonkerfield',
+    'id': 'tag:reddit.com:bonkerfield',
+    'image': {'url': 'https://styles.redditmedia.com/t5_2az095/styles/profileIcon_ek6onop1xbf41.png'},
+    'numeric_id': '59ucsixw',
+    'objectType': 'person',
+    'published': '2019-12-21T17:40:11Z',
+    'url': 'https://reddit.com/user/bonkerfield/',
+    'urls': [
+      {'value': 'https://reddit.com/user/bonkerfield/'},
+      {'value': 'https://bonkerfield.org'},
+      {'value': 'https://viewfoil.bonkerfield.org'}
+    ],
+    'username': 'bonkerfield'
+  },
+  'content': 'The ultimate purpose of Reddit will be the testing ground for passing the Turing test. Then we can all quit the internet.',
+  'id': 'tag:reddit.com:fgpvzfw',
+  'inReplyTo': [
+    {
+      'id': 'tag:reddit.com:ezv3f2',
+      'url': 'https://reddit.com/r/MachineLearning/comments/ezv3f2/p_gpt2_bert_reddit_replier_i_built_a_system_that/'
     }
   ],
-  'itemsPerPage': 1,
-  'sorted': False,
-  'startIndex': 0,
-  'totalResults': 1,
-  'updatedSince': False
+  'objectType': 'note',
```

Comment:
i think this is actually `comment`. https://activitystrea.ms/specs/json/schema/activity-schema.html#object-types

(you'll need to update the code to match.)